content/sigs/pipeline-authoring/index.adoc

```diff
@@ -0,0 +1,58 @@
+---
+layout: sig
+title: "Pipeline Authoring"
+section: sigs
+sigId: "pipeline-authoring"
+logo: /images/user.gif
+tags:
+- pipeline
+- ux
+- authoring
+leads:
+- name: "Andrew Bayer"
+  id: "abayer"
+  github: "abayer"
+participants:
+- name: "Liam Newman"
+  id: "bitwiseman"
+  github: "bitwiseman"
+  
```

Comment:
```suggestion
- name: "Devin Nusbaum"
  id: "dnusbaum"
  github: "dwnusbaum"
  
```

Reply:
Now I have a suggestion merge conflict :D

Reply:
heheh, I'll reconcile it manually.

Reply:
Just thought it would be neat to try to the new feature :D